Automattic/themes
# Added Gutenberg check to Blockbase themes

#### Changes proposed in this Pull Request:

This adds a slightly modified version of the Gutenberg Check that was added to Livro.

This version takes into account child theme-ness.  Any child of Blockbase shows the same Gutenberg Required message.  Additionally the child may express a DIFFERENT version of Gutenberg that is required and the highest version will be used for the check/message.

#### Testing

* Activate Blockbase
* See no message.
* Change version of `Requires Gutenberg:` to 13.12 or something higher than what is available: See the message.  Return the value to normal.
* Deactivate Gutenberg: See the message. (Reactivate Gutenberg)
* Activate a Blockbase theme (such as Quadrat).
* See no message.
* Deactivate Gutenberg: See the message. (Reactivate Gutenberg)
* Add a `Requires Gutenberg: 15.23` entry to Quadrat's (or whatever theme you activated)'s style.css metadata: See the message.

<img width="542" alt="image" src="https://user-images.githubusercontent.com/146530/159760130-7a94882e-62b4-4340-8a6e-f2fd5a427bec.png">

NOTE: We should make these same changes to Levro so that any child themes have the same Gutenberg requirement messaging.
